### PR TITLE
Feature/first section

### PR DESCRIPTION
**Pull request by gergana95**
Added first section html code, fonts included and images.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,6 +24,26 @@
         </nav>
     </header>
 
+    <section class="escape">
```
Comment (superKalo) [inline index.html]: Maybe a more generic name instead ... try with `.primary-section`

File: index.html [hunk 2]
```diff
@@ -22,6 +24,29 @@
         </nav>
     </header>
 
+    <section class="primary-slider group">
+            <div class="primary-slider__content left">
```
Comment (superKalo) [inline index.html]: Since `.primary-slider__content` class has `position: absolute`, probably you don't need a float left?

File: index.css [hunk 3]
```diff
@@ -61,12 +61,99 @@
...
+     transform: translateY(-50%);
+     top: 50%;
+}
+
+.primary-slider__subtitle {
+    opacity: 0.9;
+    color: #5da8ef;
+    font-size: 9px;
+    font-weight: 700;
+    line-height: 66px;
```
Comment (superKalo) [inline index.css]: Good "trick", but it's hard to measure if it's the right size. For example here - there's a difference.

File: index.css [hunk 4]
```diff
@@ -61,12 +61,99 @@
...
+    margin: 0;
+    color: #0b0c11;
+    font-family: "Playfair Display";
+    font-size: 62px;
+    font-weight: 700;
+    line-height: 64px;
+}
+
+.primary-slider__paragraph {
+    width: 47%;
```
Comment (superKalo) [inline index.css]: How did you measure that?

File: index.css [hunk 5]
```diff
@@ -61,12 +61,99 @@
...
+    font-weight: 700;
+    line-height: 66px;
+    text-transform: uppercase;
+    letter-spacing: 6px;
+}
+
+.primary-slider__title {
+    margin: 0;
+    color: #0b0c11;
+    font-family: "Playfair Display";
```
Comment (superKalo) [inline index.css]: I guess this is an obsolete declaration? Since you have a class that does the same.

File: index.css [hunk 6]
```diff
@@ -61,12 +61,99 @@
...
+    font-size: 62px;
+    font-weight: 700;
+    line-height: 64px;
+}
```
Comment (superKalo) [inline index.css]: Discuss a bit the "vertical rhythm" (margins in only 1 direction).

File: index.css [hunk 7]
```diff
@@ -61,12 +61,99 @@
...
 
+/* section primary-slider */
+
+.primary-slider {
+    position: relative;
+    padding-top: 43px;
+}
+
+.primary-slider__content {
+     /*linear-gradient(to right, rgba(255, 255, 255, 0.2) 0%, rgba(255, 255, 255, 0) 100%);*/
```
Comment (superKalo) [inline index.css]: If it doesn't do anything at all, kill it.

File: images/right-arrow.svg [hunk 8]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="11" height="19" viewBox="0 0 11 19"><path fill="#ebebeb" d="M9.85 9.64L.92 1.14a.55.55 0 0 0-.76 0 .49.49 0 0 0 0 .73L8.71 10 .16 18.13a.5.5 0 0 0 0 .72.55.55 0 0 0 .76 0l8.92-8.5a.49.49 0 0 0 0-.71z"/></svg>
```
Comment (superKalo) [inline images/right-arrow.svg]: Since you're using them directly in the HTML, why you're committing the resources too?

File: index.css [hunk 9]
```diff
@@ -61,12 +61,99 @@
     color: #5da8ef;
 }
 
+/* section primary-slider */
+
+.primary-slider {
+    position: relative;
+    padding-top: 43px;
```
Comment (superKalo) [inline index.css]: Discuss a bit the "vertical rhythm" (margins in only 1 direction).

File: index.css [hunk 10]
```diff
@@ -61,12 +61,99 @@
...
+.slider-controls__right-arrow:hover {
+    fill: #5da8ef;
+}
+
+.slider-controls__right-arrow {
+    margin-left: 15px;
+}
+
+.font-secondary {
+    font-family: "Playfair Display", Arial, Tahoma, sans-serif;
```
Comment (superKalo) [inline index.css]: See http://maconprinting.com/serif-versus-san-serif-fonts => this is not sans-serif font => probably another fallback fonts are more appropriate.

File: index.css [hunk 11]
```diff
@@ -61,12 +61,99 @@
...
+}
+
+.primary-slider__image {
+    box-shadow: -7px 19px 20px 5px darkgrey;
+}
+
+
+.slider-controls {
+    position: absolute;
+    top: 93%;
```
Comment (superKalo) [inline index.css]: Are there any, more convenient properties you can use to align it at the bottom?

File: index.css [hunk 12]
```diff
@@ -61,12 +61,99 @@
...
+}
+
+.slider-controls__left-arrow {
+    margin-right: 15px;
+}
+
+.slider-controls__left-arrow:hover,
+.slider-controls__right-arrow:hover {
+    fill: #5da8ef;
+}
```
Comment (superKalo) [inline index.css]: There is an issue with the hover effect.

File: index.html [hunk 13]
```diff
@@ -22,6 +24,29 @@
...
+                <span class="primary-slider__subtitle font-primary">your</span>
+                <h1 class="primary-slider__title font-secondary">Beautiful Escape</h1>
+                <p class="primary-slider__paragraph font-tertiary">One of the greatest things about the sport of surfing is that you need only three things: your body, a surfboard, and a wave.</p>
+            </div>
+
+            <img class="primary-slider__image right" src="images/img.jpg" alt="Sea image">
```
Comment (superKalo) [inline index.html]: **

File: index.css (hunk 5 above)
Comment (gergana95) [reply index.css]: removed

File: index.html (hunk 2 above)
Comment (gergana95) [reply index.html]: removed

File: index.css (hunk 7 above)
Comment (gergana95) [reply index.css]: removed

**Comment by superKalo** [comment]
Okay, now it looks awesome! 👍

**Pull request merged**